### PR TITLE
New Language: Persian - Lang 36

### PR DESCRIPTION
**Pull request by tandy-1000**
#### Database Migration
No

#### Description
Support importing Iranian movies with the correct Persian language label.

### CODE REVIEW

File: src/NzbDrone.Core/Parser/IsoLanguages.cs [hunk 1]
```diff
@@ -38,6 +38,7 @@ public static class IsoLanguages
                                                                new IsoLanguage("ro", "", "ron", "Romanian", Language.Romanian),
                                                                new IsoLanguage("pt", "br", "", "Portuguese (Brazil)", Language.PortugueseBR),
                                                                new IsoLanguage("ar", "", "ara", "Arabic", Language.Arabic)
```
Comment (RobinDadswell) [inline src/NzbDrone.Core/Parser/IsoLanguages.cs:40]: missing comma

**Comment by bakerboy448** [comment]
Build failed

you'll also need to rebase and change the # once the several other outstanding language PRs are merged

File: src/NzbDrone.Core.Test/Languages/LanguageFixture.cs [hunk 2]
```diff
@@ -43,7 +43,9 @@ public class LanguageFixture : CoreTest
                 new object[] { 28, Language.Thai },
                 new object[] { 29, Language.Bulgarian },
                 new object[] { 30, Language.PortugueseBR },
-                new object[] { 31, Language.Arabic }
+                new object[] { 31, Language.Arabic },
+                new object[] { 32, Language.Persian }
```
Comment (RobinDadswell) [inline src/NzbDrone.Core.Test/Languages/LanguageFixture.cs:47]: please check the number vs other pre-existing language PRs

**Review by RobinDadswell — CHANGES_REQUESTED** [review]

**Comment by bakerboy448** [comment]
Test cases for release parsing?

does "Persian" actually show up in the wild for releases

File: src/NzbDrone.Core/Parser/IsoLanguages.cs (hunk 1 above)
Comment (tandy-1000) [reply src/NzbDrone.Core/Parser/IsoLanguages.cs]: Oops, fixed.

File: src/NzbDrone.Core.Test/Languages/LanguageFixture.cs (hunk 2 above)
Comment (tandy-1000) [reply src/NzbDrone.Core.Test/Languages/LanguageFixture.cs]: I incremented the number to take #6385 into account.

**Comment by tandy-1000** [comment]
> Test cases for release parsing?
> 
> does "Persian" actually show up in the wild for releases

No, I tried to have a quick look for what Iranian / Persian films come up as, but I can't seem to find any as shown in #6385.

**Comment by bakerboy448** [comment]
- Lang 32 in use by https://github.com/Radarr/Radarr/pull/6385 
 - Lang 33 in use by https://github.com/Radarr/Radarr/pull/6415
 - Lang 34 in use by https://github.com/Radarr/Radarr/pull/6522
 - Lang 35 in use by https://github.com/Radarr/Radarr/pull/6522
use lang 36

**Review by bakerboy448 — DISMISSED** [review]
update language # & etc for conflicts

**Comment by tandy-1000** [comment]
I updated the language number, let me know if anything else is required.

**Comment by RobinDadswell** [comment]
@tandy-1000  instead of a merge commit can you rebase it onto develop?

**Comment by tandy-1000** [comment]
> @tandy-1000 instead of a merge commit can you rebase it onto develop?

Oops sorry, just done that

**Comment by bakerboy448** [comment]
Try again - it looks like you merged again.

**Comment by tandy-1000** [comment]
> Try again - it looks like you merged again.

how about now?

**Comment by bakerboy448** [comment]
Again a it appears you're merging the branch  in instead of rebasing. Or perhaps you rebased on your forks develop rather than origin develop?

you should not have 82 commits in this PR - you should have one.

the builds are also now failing

**Comment by tandy-1000** [comment]
I've mucked it up :/ I'll have a look at doing that..

**Review by RobinDadswell — CHANGES_REQUESTED** [review]
Still a merge commit

**Comment by tandy-1000** [comment]
> Still a merge commit

Could you provide some guidance on how I should go about this? This time I reset to my last commit, then used this [guide](https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a) to rebase.. I'm not sure what else to do

**Comment by RobinDadswell** [comment]
> > Still a merge commit
> 
> Could you provide some guidance on how I should go about this? This time I reset to my last commit, then used this [guide](https://medium.com/@topspinj/how-to-git-rebase-into-a-forked-repo-c9f05e821c8a) to rebase.. I'm not sure what else to do

Good news is it looks like it's now fixed! One last thing - one further thing is can you do an interactive rebase, squash all the commits into one?

**Comment by bakerboy448** [comment]
Seems the latest force push did it as far as your merge/rebase hell went.  Would just need the commits squashed down into a single meaningful commit

`New: Add Persian Language Support` or something like that 

PS check out GitExtensions for a GUI and less git cmds

**Comment by tandy-1000** [comment]
Oh great, thanks! I'll do that.
I was originally using VSCodium's git GUI... It wasn't too helpful

**Comment by tandy-1000** [comment]
Done! :)

**Pull request closed without merging**